### ISSUE TITLE
Repeat Play Button

### ISSUE THREAD

**Issue by McCoyCOREUSA** [xLightsSequencer/xLights#4877]
v2024.16.2

I've noticed the REPEAT PLAY button isn't available when a time selection is selected. I can only click the PLAY button. Once it plays once, then I can click the repeat play button. 

Seems the repeat play button should be available when the play button is available. Perhaps even moved next to play?

**Comment by derwin12** [comment]
Are you sure?
![image](https://github.com/user-attachments/assets/98db425b-2013-4aa2-98ee-6c53f988ea8d)

**Comment by McCoyCOREUSA** [comment]
Yes. I create a section to play. Repeat button is unavailable. Once I play it once (play button), then the repeat button becomes available.
![Screenshot 2024-10-12 114159](https://github.com/user-attachments/assets/3479eab0-a050-44f5-878d-efd3e49283cc)

**Comment by McCoyCOREUSA** [comment]
I'll try to catch you in the Zoom Room and show you.
